homeassistant/components/calendar/trigger.py

```diff
@@ -81,10 +81,10 @@ def async_detach(self) -> None:
 
     async def _fetch_events(self, now: datetime.datetime) -> None:
         """Update the set of eligible events."""
-        start_date = now
-        end_date = now + UPDATE_INTERVAL
-        _LOGGER.debug("Fetching events between %s, %s", start_date, end_date)
-        events = await self._entity.async_get_events(self._hass, start_date, end_date)
+        last_endtime = now
```

Comment:
Would it make sense to just call the parameter `last_endtime` so we don't have to do this assignment?

Reply:
👍🏼